scripts/powershell/sub-cleanup/exceptions.txt

```diff
@@ -0,0 +1,5 @@
+bterlson-testbed
+js-keyvault-tests-danrodri
```

Comment:
Why do these other 4 RGs need an exception instead of being renamed?

Reply:
If we do need exceptions, would it be better to use say a tag on the RG, instead of a text file in this repo?  This way RGs can be granted an exception directly in the Azure Portal instead of requiring a code change.

Reply:
Mmm... I like the tags suggestion.  I'll change it to use tags.  It elminates the possiblity of someone forgetting to use the exception file when they run it.

The other RGs had some complexities that made it difficult for them to transfer resources to another group.  I told them all that these exceptions could be made, but all new RGs needed to follow the patter or they'd be deleted automatically.

Reply:
I like that option better too.